### PR TITLE
10 miscellaneous

### LINKED ISSUE #10: Miscellaneous
- [x] #15
- [x] #16

### PR DESCRIPTION
**Pull request by priyapal2000**
In this epic have two task that is menu bar and career object.
closes #10

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      color: white;
+      text-decoration: none;
+    }
+    .menu ul li a:hover
+    {
+      color: red;
+    }
+body,
+h1,
+p {
```
Comment (ajayg2808) [inline css/index.css]: Do not use tag/ element selectors

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    .menu ul li a
+    {
+      color: white;
+      text-decoration: none;
+    }
+    .menu ul li a:hover
+    {
+      color: red;
+    }
+body,
```
Comment (ajayg2808) [inline css/index.css]: Do not add CSS for body tag

File: index.html [hunk 3]
```diff
@@ -10,9 +10,38 @@
...
+        <nav>
+            <ul>
+                <li><a href="#">Personal Details</a></li>
+                <li><a href="#">Educational Details</a></li>
+                <li><a href="#">Skill Set</a></li>
+                <li><a href="#">Courses & Certifications</a></li>
+                <li><a href="#">Extra Curricular</a></li>
+                <li><a href="#">Projects</a></li>
+                <li><a href="#">Social Platform</a></li>
+                <li><a href="#">Contact Me</a></li>
```
Comment (ajayg2808) [inline index.html]: Add menu links. Add section ID's as reference link.

**Review by ajayg2808 — CHANGES_REQUESTED** [review]
Refer for menu links: https://www.w3schools.com/html/html_links_bookmarks.asp

File: css/index.css (hunk 2 above)
Comment (priyapal2000) [reply css/index.css]: Remove the body tag in CSS file is done.

File: css/index.css (hunk 1 above)
Comment (priyapal2000) [reply css/index.css]: changes done selectors tag.

File: index.html (hunk 3 above)
Comment (priyapal2000) [reply index.html]: Successfully added id's .

**Review by ajayg2808 — APPROVED** [review]
Always follow code formatting and naming conventions.

**Pull request merged**